### DISCUSSION TITLE
Filter by music provider in Artists, Albums, Tracks, etc. sections.

### DISCUSSION THREAD

**Discussion by freedomanonyme** [music-assistant/support#2064] [Feature requests and ideas]
It could be nice to enable a filter that give me the option to see only YoutubeMusic and/or Spotify and/or local music, etc.

**Comment by X-dark** [comment]
The only partial solution I could find is to disable provider(s) you don't want to see and their cover will be greyed out. So album from remaining provider(s) will be easier to spot.

**Comment by clang88** [comment]
This seems like a really cool feature. I sometimes specifically know I want to just browse my local results, or my personal spotify and not the one from my partner. So having the option to filter on provider on all lists seems like a really good thing to have. :)

**Comment by OzGav** [comment]
Duplicate of https://github.com/orgs/music-assistant/discussions/1538. Please vote over there
